client/src/components/Button/ThankButton.js

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import styled from "styled-components";
+import { mq, theme } from "constants/theme";
+
+const primaryColor = theme?.colors?.royalBlue;
+const whiteColor = theme?.colors?.offWhite;
+
+const ThanksButton= styled.div`
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 8px 38px;
+
+position: absolute;
+width: 152px;
+height: 40px;
+left: 1083px;
+top: 305px;
+
+background: #FFFFFF;
+
+border: 1px solid #425AF2;
+box-sizing: border-box;
+border-radius: 46px;
+width: 152px;
+height: 40px;
+top: 305px;
+left: 1083px;
+radius: 46px;
+padding: 8px, 38px, 8px, 38px;
+cursor: pointer;
+
+@media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    display: none;
+  }
+ `
+
+const ThankButton = () => {
+    return(
+        <>
+        <ThanksButton>♡Thanks</ThanksButton>
+        </>
+    )
+}
+
+export default ThankButton
+
+
+
+// import { Modal, Button } from "antd";
```

Comment:
I think we can get rid of the commented out code, if its a relic we don't need it in here, and if we ever DO need to go back and reference it we can just get it from and an older version

Reply:
ok!